library/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateLogAnalyticsWorkspaceStep.java

```diff
@@ -105,7 +105,7 @@ protected void createResource(FlightContext context, ArmManagers armManagers) {
                 context
                     .getInputParameters()
```

Comment:
Unrelated to this PR, but doesn't this need to use the `ParametersResolver`?

Reply:
Do you mean that `ParameterResolver` isn't used here?